index.js

```diff
@@ -59,13 +61,25 @@ function lint(input, config, webpack) {
     res = engine.executeOnText(input, resourcePath, true)
 
     // Save new results in the cache
-    if (config.cache) {
+    if (config.cache && cachePath) {
       cache[resourcePath] = {
         hash: inputMD5,
         rules: rulesHash,
         res: res,
       }
-      fs.writeFileSync(cachePath, JSON.stringify(cache))
+      var cacheJson = JSON.stringify(cache)
+      try {
+        safeWriteCache(cachePath, cacheJson)
+      } 
+      catch (e) {
+        try {
+          cachePath = cacheFallback
+          safeWriteCache(cacheJson)
+        } 
+        catch (e) {
+          cache = false
```

Comment:
Why are you using `false` here? Does it have a special meaning compared to `null` elsewhere?